sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyOperationResult.java

```diff
@@ -23,24 +23,81 @@
     private Base64Url result;
 
     /**
-     * Get the kid value.
+     * Initialization vector used for some cryptographic operations.
+     */
+    @JsonProperty(value = "iv", access = JsonProperty.Access.WRITE_ONLY)
+    private Base64Url iv;
+
+    /**
+     * Authentication tag used for some cryptographic operations.
+     */
+    @JsonProperty(value = "tag", access = JsonProperty.Access.WRITE_ONLY)
+    private Base64Url authenticationTag;
+
+    /**
+     * Additional authenticated data used for some cryptographic operations.
+     */
+    @JsonProperty(value = "aad", access = JsonProperty.Access.WRITE_ONLY)
+    private Base64Url additionalAuthenticatedData;
+
+    /**
+     * Get the key identifier.
      *
-     * @return the kid value
+     * @return The key identifier.
      */
     public String getKid() {
         return this.kid;
     }
 
     /**
-     * Get the result value.
+     * Get the result.
      *
-     * @return the result value
+     * @return The result.
      */
     public byte[] getResult() {
         if (this.result == null) {
             return new byte[0];
         }
+
         return this.result.decodedBytes();
     }
 
+    /**
+     * Get the initialization vector.
+     *
+     * @return The initialization vector.
+     */
+    public byte[] getIv() {
+        if (this.iv == null) {
+            return new byte[0];
```

Comment:
Given that `new byte[0]` is commonly used, and is effectively immutable, we should make this a constant on the class